packages/playground-server/src/resources/user/processor.ts

```diff
@@ -69,12 +70,16 @@ export default class UserProcessor extends OperationProcessor {
       tags: { userId: user.id, endpoint: "createAccount" }
     });
 
-    NodeWrapper.createStateChannelFor(nodeAddress as string).then(
-      async (result: Node.CreateChannelResult) => {
+    const { transactionHash } = await NodeWrapper.createStateChannelFor(
+      nodeAddress as string,
+      async (result: NodeTypes.CreateChannelResult) => {
         await bindMultisigToUser(newUser, result.multisigAddress);
       }
     );
 
+    newUser.attributes.transactionHash = transactionHash;
+    await bindTransactionHashToUser(newUser, transactionHash);
```

Comment:
Is it a bad idea to set the `transactionHash` attribute on the front-end User instance _inside_ the `bindTransactionHashToUser` function even though this function is only concerned with setting the property on the backend DB?

Reply:
It's a matter of separation of concerns. `bindTransactionHashToUser` (as well as `bindMultisigToUser`) should be better named. Something like `saveTransactionHash`, to make it clear that the function doesn't deal with how the API responds.

Reply:
Not urgent, just a passing thought.